config/main.py

```diff
@@ -753,23 +753,23 @@ def _restart_services(config_db):
 
 def interface_is_in_vlan(vlan_member_table, interface_name):
     """ Check if an interface  is in a vlan """
-    for _,intf in vlan_member_table.keys():
+    for _,intf in list(vlan_member_table.keys()):
```

Comment:
>_, [](start = 8, length = 2)

Add a blank after `,`

Reply:
Added a space here and after all other commas in the file which were not followed by a space.